extensions/amp-infinite-scroll/0.1/test/validator-amp-infinite-scroll.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
This is missing the appropriate license with 2019 as year.

Should also include comment explaining what this file is testing and also each test case within the document. See [this as an example](https://github.com/ampproject/amphtml/blob/master/extensions/amp-autocomplete/0.1/test/validator-amp-autocomplete.html).

Reply:
@honeybadgerdontcare thank you, validation rules should be fixed now